_typos.toml

```diff
@@ -0,0 +1,263 @@
+[files]
```

Comment:
This is the config file for the awesome CLI tool [typos](https://github.com/crate-ci/typos) I've used to create this PR. 

I will remove it before merging this PR (and possibly other PRs "split out" from this, if we go with the "batch" approach).

N.B. I've done a lot of manual work apart from using the typos cli tool, which have taken many hours, e.g. git added some true positives of some files, but ignored some false positives in the same files, and then "typos ignored" those files. The implications of this is that if you were to run the same version of typos with this config file, **you would NOT get the same result, some typos would be missed**. I did it like this since many files contained lots of specific false positives, where it made no sense to add those strings to extended identifiers list.